### PR TITLE
Polishing fixes of lab4

### PR DESCRIPTION
**Pull request by dahaic**

### CODE REVIEW

File: 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc [hunk 1]
```diff
@@ -158,6 +172,25 @@ Now, we will add an Ansible task or tasks for this rule below the header in `sha
...
+# disruption = low
+
+- name: configure timeout
+  lineinfile:
+      create: yes
+      dest: /etc/profile
+      regexp: "^#?TMOUT"
+      line: "TMOUT=1800"
+----
+
```
Comment (matejak) [inline 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc]: Is it now time to close the file?

And could you please add the "Use Ctrl+Shift+V" to paste into the editor hint?

File: 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc [hunk 2]
```diff
@@ -311,52 +354,64 @@ To use our Playbook on your machine (on a local host), replace the placehoder st
...
 Notice that `TMOUT=1800` is at the end of the file!
 
 The biggest advantage of using Ansible tasks in ComplianceAsCode is that it gets integrated with the SCAP content and also the HTML report and in the HTML guide as well.
 
+Switch to the console view, open the terminal.
 Run the following command to open the HTML guide for OSPP profile for Red Hat Enterprise Linux 8 in your Firefox web browser:
 
 ----
-$ firefox build/guides/ssg-rhel8-guide-ospp.html
+[... ~]$ firefox /home/lab-user/labs/lab4_ansible/build/guides/ssg-rhel8-guide-ospp.html
```
Comment (matejak) [inline 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc]: Please consider adding something like "use this terminal command or navigate to the OSPP guide in the same way as you did in previous exercises".

File: 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc [hunk 3]
```diff
@@ -382,7 +437,7 @@ Although the directory name is `roles`, and the files are called roles, these fi
 Check the contents of the OSPP profile Playbook in your editor and verify that a task for rule `accounts_tmout` is there among all the other tasks.
 
 ----
-$ gedit build/roles/ssg-rhel8-role-ospp.yml
+[... lab4_ansible]$ nano build/roles/ssg-rhel8-role-ospp.yml
 ----
 
 Keep in mind that even if the comments in the headers say that it is an Ansible Role, it is not an Ansible Role, it is an Ansible Playbook.
```
Comment (matejak) [inline 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc]: The exercise somehow vanishes into nothingness - could you think of some sweet ending?

Comment (dahaic) [reply 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc]: I have tried to put something small at the end.

File: 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc (hunk 2 above)
Comment (dahaic) [reply 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc]: did so, thanks for the suggestion!

File: 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc (hunk 1 above)
Comment (dahaic) [reply 2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc]: It's written few lines before the snippet. Pasting hint added

**Pull request merged**